test/benchmark-queue-work.c

```diff
@@ -0,0 +1,47 @@
+#include "task.h"
+#include "uv.h"
+
+static int done = 0;
+static unsigned events = 0;
+
+static unsigned fastrand(void) {
+  static unsigned g = 0;
+  g = g * 214013 + 2531011;
+  return g;
+}
+
+static void work_cb(uv_work_t *req) { 
+  fastrand();
+}
+
+static void after_work_cb(uv_work_t *req, int status) {
+  static uv_work_t handle;
+
+  events++;
+  if (!done)
+    ASSERT(0 == uv_queue_work(req->loop, &handle, work_cb, after_work_cb));
```

Comment:
Can you use the new style `ASSERT_EQ` macro here and elsewhere?

Also, you can re-use `req` here, you don't need `handle`. Incidentally, `handle` is not an apt name because it's not a handle.